README.md

```diff
@@ -1 +1,58 @@
 # Treepat
+
+Treepat is a language to recognize patterns in trees in a similar way as regular expressions recognize patterns in 
+strings.Treepat includes analogous operators to regex union, concatenation, and closure, which are extended to the 
+notion of trees.
+
+## Installation
+
+Follow these steps to install the project.
+
+### Prerequisites
+
+Make sure you have [maven](https://maven.apache.org/) installed. To make sure you have it installed run:
+```bash
+mvn --vesion
+```
+
+### Compile
+To run the project must first compile the source code using the following maven command:
+```bash
+mvn compile 
+```
+
+### Test
+To make sure that everything was installed correctly run the tests by running:
+```bash
+mvn test 
+```
+
+### Run
+The repository contains a sample input files. To run the interpreter use the following command:
+```bash
+mvn exec:java -Dexec.mainClass="Main" -Dexec.args="test.tp test.tef" 
+```
+Note that `test.tp` is a file that contains the Treepat expression and `test.tef` contains the target tree to find 
+patterns.
```

Comment:
This might be the biggest nitpick on this review, but I don't think "`test.tef` contains the target tree to find 
patterns." is well phrased. I'm thinking of something along the lines of
```suggestion
Note that `test.tp` is a file that contains the Treepat expression and `test.tef` contains the target tree where the patterns will be matched.
```
I'm not entirely sure of this one, however.